src/ui/nqqsettings.cpp

```diff
@@ -0,0 +1,69 @@
+#include "include/nqqsettings.h"
+
+#ifdef QT_DEBUG
+#include <QDebug>
+#endif
+
+void NqqSettings::ensureBackwardsCompatibility()
+{
+    QSettings s;
+
+    const QStringList keys = s.allKeys();
+
+    //If this string is in the settings we either have the new ini settings
+    if(keys.contains("BackwardsCompatible"))
+            return;
```

Comment:
I think we should take this opportunity to add, within the settings, a special key which represents the version of Notepadqq that wrote that file. The value should be updated each time Nqq is started.

So, instead of `keys.contains("BackwardsCompatible")`, we will check if `NotepadqqVersion` key is present or not.

I think it will come useful in the future, for situations like this ;)